CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -119,6 +119,13 @@ public bool TryGetProperty(long objectId, string name, out object result, out st
                 throw new InvalidOperationException(string.Format("Property {0} not found on Object of Type {1}", name, obj.Value.GetType()));
             }
 
+            if (property.JsObject != null)
+            {
+                obj = property.JsObject.Bind();
+                result = !obj.IsNull && obj.Value.GetType().IsArray ? obj.Value : obj;
```

Comment:
Can you clarify, your second sentence isn't clear.

Code needs comment to document behavior.

Somewhat sounds like array should be treated differently and the model should include first class support, to make the behavior clearer.